cli/tests/integrations/test_package.py

```diff
@@ -671,6 +669,46 @@ def test_search_table():
     assert stderr == b''
 
 
+def test_search_ends_with_wildcard():
```

Comment:
We've reverted commits maybe a couple of times in the 4 months I've been here. But i find a small separate formatting/design issue to fix in almost every PR I write. The extra context switching cost is not worth any git revert benefit. I would be very strongly against this policy.